tools/run_tests/run_interop_tests.py

```diff
@@ -1205,6 +1215,13 @@ def aggregate_http2_results(stdout):
   num_failures, resultset = jobset.run(jobs, newline_on_success=True,
                                        maxjobs=args.jobs,
                                        skip_jobs=args.manual_run)
+  if args.bq_result_table and resultset:
+    # Transient errors in uploading results to BQ should not cause the build to fail
+    try:
+      upload_results_to_bq(resultset, args.bq_result_table, args, jobset.platform_string())
+    except Exception as e:
```

Comment:
same issue as the other PR - except in this case we will at least see the logs if uploading to bigquery fails. Regardless what we do, we should stay consistent with what run_tests.py does.

Reply:
Please remove the try except, things should work fine with https://github.com/grpc/grpc/pull/12839.

Reply:
Done